Demo/Demo/FeatureViewControllers/CardDemoViewController.swift

```diff
@@ -125,11 +125,12 @@ class CardDemoViewController: FeatureBaseViewController, UITextFieldDelegate {
         let config = CoreConfig(clientID: DemoSettings.clientID, environment: DemoSettings.environment.paypalSDKEnvironment)
         let cardClient = CardClient(config: config)
 
-        cardClient.approveOrder(orderID: orderID, card: card) { result in
+        let cardOrder = CardOrder(orderID: orderID, card: card)
+        cardClient.approve(cardOrder) { result in
```

Comment:
Looking at the changes from the merchant perspective, this naming feels pretty confusing. I wonder if `CardRequest` would be a better name than `CardOrder`? To me, order implies that they would use this to create the order, but in reality they are just passing us an order ID (having created an order outside of this request) and a card object. I'm not tied to `CardRequest` either, just trying to think through if there may be a more descriptive name for this. Thoughts?

Reply:
Yeah I left these in a single commit so they'd be easy to revert, but I wanted to try to align naming more with [this doc](https://www.swift.org/documentation/api-design-guidelines/#omit-needless-words) to see what it looks like. The example they give in the More Detail section shows the method name as a verb and the first argument is like a noun/subject, so it reads to the merchant as "card client approve card order." To that point, "card client approve card request" would also work.

It may be out of scope for this PR but definitely wanted to try it out to see if we could get more expressiveness out of the API.

Reply:
I think it makes sense to revert this for now and keep it in mind down the line. For now, the `CardRequest` object should help us extend the API without causing a breaking change to the method signature.